exp/controllers/azuremangedcontrolplane_reconciler.go

```diff
@@ -36,13 +36,15 @@ import (
 	"sigs.k8s.io/cluster-api-provider-azure/cloud/scope"
 	"sigs.k8s.io/cluster-api-provider-azure/cloud/services/groups"
 	"sigs.k8s.io/cluster-api-provider-azure/cloud/services/managedclusters"
+	"sigs.k8s.io/cluster-api-provider-azure/cloud/services/virtualnetworks"
 	infrav1exp "sigs.k8s.io/cluster-api-provider-azure/exp/api/v1alpha3"
 )
 
 // azureManagedControlPlaneReconciler are list of services required by cluster controller
 type azureManagedControlPlaneReconciler struct {
 	kubeclient         client.Client
 	managedClustersSvc *managedclusters.Service
+	vnetSvc            *virtualnetworks.Service
```

Comment:
this should be of type azure.Service

we're only using `*managedclusters.Service` as a workaround until https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/757 is done

Reply:
I don't want a reconciler here? I actually want to be able to List vnets, which the vnet-specific service exposes while the reconciler interface doesn't?

Reply:
Is it still being used? I don't see the vnetSvc in use anymore

Reply:
whoops yeah, I'll remove this